apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -598,21 +597,50 @@ export function RequiredFirmwareUpdate({
 }) {
   const lastSeenDevice: DeviceModelInfo | null | undefined = useSelector(lastSeenDeviceSelector);
 
-  const isUsbFwVersionUpdateSupported =
-    lastSeenDevice && isFirmwareUpdateVersionSupported(lastSeenDevice.deviceInfo, device.modelId);
-
-  const usbFwUpdateActivated = Platform.OS === "android" && isUsbFwVersionUpdateSupported;
+  const usbFwUpdateActivated = !!lastSeenDevice;
 
   const deviceName = getDeviceModel(device.modelId).productName;
 
   const isDeviceConnectedViaUSB = device.wired;
 
+  const track = useTrack();
+
   // Goes to the manager if a firmware update is available, but only automatically
   // displays the firmware update drawer if the device is already connected via USB
   const onPress = () => {
-    navigation.navigate(NavigatorName.MyLedger, {
-      screen: ScreenName.MyLedgerChooseDevice,
-      params: { device, firmwareUpdate: isDeviceConnectedViaUSB },
+    track("button_clicked", {
+      button: "OpenMyLedger",
+      page: "Update_OS_To_Continue",
+    });
+    navigation.reset({
```

Comment:
Can we not just navigate ? (using the same navigators and routes that you added)